addons/kommander/1.2/kommander.yaml

```
@@ -9,7 +9,7 @@ metadata:
     # This was originally added to support the PVC's needed for the Kubecost subcomponent.
     kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "1.2.0-28"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.2.0-29"
     appversion.kubeaddons.mesosphere.io/kommander: "1.2.0-rc.4"
```

```suggestion
    appversion.kubeaddons.mesosphere.io/kommander: "1.2.0-rc.5"
```

theres a rc4 release already…